cypress/e2e/process.cy.js

```diff
@@ -0,0 +1,166 @@
+import ProcessesPageObjects from "../pageObjects/ProcessesPage";
+import TenureRequestDocsPageObjects from "../pageObjects/tenureRequestDocumentsPage";
+import TenureReviewDocsPageObjects from "../pageObjects/tenureReviewDocumentsPage";
+import PersonPageObjects from "../pageObjects/personPage";
+import PropertyPageObjects from "../pageObjects/propertyPage";
+import TenurePageObjects from "../pageObjects/tenurePage";
+import { seedDatabase, seedDatabaseWithTenure } from "../helpers/DbHelpers";
+
+
+const processesPage = new ProcessesPageObjects();
+const personPage = new PersonPageObjects();
+const propertyPage = new PropertyPageObjects();
+const tenurePage = new TenurePageObjects();
+const tenureReqDocsPage = new TenureRequestDocsPageObjects();
+const tenureReviewDocsPage = new TenureReviewDocsPageObjects();
+
+const tags = ['@process', '@common', '@root', '@authentication', '@personal-details']
+
+describe('Processes menu', {tags: tags}, ()=> {
+    beforeEach(()=> {
+        cy.login()
+        seedDatabaseWithTenure(true);
+    })
+
+    it('should naviage to processes on person, tenure and asset pages', {'tags': '@SmokeTest'},()=> {
+        cy.getPersonFixture().then(({ id: personId }) => {
+            personPage.visit(personId);
+            personPage.newProcess().click()
+            processesPage.pageTitle()
+            processesPage.processesMenuList().should('be.visible')
+            cy.contains('Back').click()
+
+            cy.get(":nth-child(1) > .govuk-link").click({force: true});

```

Comment:
This ideally would be a `datatest-id`, POM method, or nth item of some non-style purpose class...
Because it's not clear what UI item is selected here. It may be a process list, or it may not :man_shrugging: hard to say w/o having awareness of all page's components.

Reply:
couldn't find the datatest-id so i've added a comment to show where its going